system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -129,7 +129,7 @@
       }
     }
 
-    .tippy-top-icon {
+    .rich-icon {
       position: absolute;
```

Comment:
Might be okay to just put a `background-color: white` here? Checking with design…

<img width="128" alt="image" src="https://user-images.githubusercontent.com/438537/30250736-a1ea1a54-9608-11e7-8afe-e6622e0253ae.png">